ocean_albedo.F90

```diff
@@ -399,6 +406,63 @@ subroutine compute_ocean_albedo_new (ocean, coszen, albedo_vis_dir, &
   albedo_nir_dif = 0.06
 endif
 
+where (.not.ocean)
+  albedo_vis_dir = 0.0
+  albedo_vis_dif = 0.0
+  albedo_nir_dir = 0.0
+  albedo_nir_dif = 0.0
+end where
+
+if (ocean_albedo_option == 6) then
+   
+   if(present(flux_u) .and. present(flux_v)) then
+   
+!
+!  Add white cap coverage as a function of wind speed WC = 0.0397*U10^1.59 converted from % to ratio from:
+!   Salisbury, D. J., Anguelova, M. D., and Brooks, I. M.:
+!   Global Distribution and Seasonal Dependence of Satellite-based Whitecap Fraction,
+!   Geophys. Res. Lett., 41, 1616–1623, https://doi.org/10.1002/2014GL059246, 2014. 
+!
+!   and visible albedo of White Caps = 0.5 from:
+!   Stabeno, P. J., & Monahan, E. C. (1986). The influence of whitecaps on the albedo
+!   of the sea surface. In Oceanic Whitecaps (pp. 261-266). Springer, Dordrecht.
+!
+   where(ocean)
+       ua=flux_u ! Note rough conversion from stress to speed
+       va=flux_v ! Note rough conversion from stress to speed
+   endwhere
+
+   call invert_tau_for_du(ua, va) ! Note rough conversion from stress to speed
+
+   where(ocean)
+       u10 = sqrt(ua*ua + va*va)
+   endwhere
+
+  where (coszen .ge. 0.0)
+    albedo_vis_dir = 0.026/(coszen**1.7+0.065)                  &
+                    +0.15*(coszen-0.10)*(coszen-0.5)*(coszen-1.0)
+  elsewhere
+    albedo_vis_dir = 0.4075 ! coszen=0 value of above expression
+  endwhere
+  white_cap_coverage=0.000397*u10**1.59
+  albedo_vis_dif = 0.06*(1-white_cap_coverage)+0.5*white_cap_coverage
+  albedo_nir_dif = 0.06
+  albedo_nir_dir = albedo_vis_dir
+  albedo_vis_dir = albedo_nir_dir*(1-white_cap_coverage)+max(albedo_nir_dir,0.5)*white_cap_coverage
+endif
+
+else
```

Comment:
@john.dunne@noaa.gov I think this else block is misplaced. What is the intention here? What is the code supposed to do if ocean_albedo_option == 6 and flux_u,flux_v are not present?